portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,21 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomTVQuote() {
```

Comment:
Could you add a function comment? (I know the function name is pretty descriptive but it's good practice to comment your functions giving a little description of what they do), maybe something like: "selects a tv quote at random and displays it on the page"

Reply:
Added in the function name comment "Adds a random tv quote to the welcome home page."